### DISCUSSION TITLE
Add Bedrock Mods/packs support

### DISCUSSION THREAD

**Discussion by Zac0511** [modrinth/.github#246] [Feature requests]
Modrinth currently only have **Java things**, but Bedrock can also load custom mods/packs :

- Addon (mods) : You can load `.mcaddon` files by clicking them in explorer, and the game installs it for you.
- Ressource packs : You can load `.mcpack` files by clicking them in explorer, and the game installs it for you.

So yeah i was thinking that Modrinth could have a menu for Bedrock things.

**Comment by blryface** [comment]
That's already planned

**Reply by Zac0511** [reply]
Oh i didn't knowed that
